lib/generators/templates/atmosphere.rb

```diff
@@ -107,6 +107,11 @@
   #
   # config.metrics_store = Atmosphere::Monitoring::NullMetricsStore.new
 
+  # Defines the client that provides information about the network id
+  # for instantiated vm.
+  # Possible values:
+  #   - Atmosphere::NicProvider::NullNicProvider.new
```

Comment:
I believe this line is missing:

```ruby
nic_provider_class = 'Atmosphere::NicProvider::NullNicProvider`
```

and in the line above `.new` should not be there